warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -215,16 +237,32 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                 browser_details = selenium_Utils.\
                     get_browser_details(browser, datafile=self.datafile, **arguments)
             if browser_details is not None:
+                headless_mode_pattern = re.compile(r'y(es)?$', flags=re.IGNORECASE)
                 if type == "firefox":
                     ff_profile = self.browser_object.\
                         set_firefoxprofile(proxy_ip, proxy_port)
+                if headless_mode_pattern.match(str(headless_mode).strip()):
+                    try:
+                        from pyvirtualdisplay import Display
+                        display = Display(visible=0, size=(1024, 768))
+                        display.start()
+                        pNote("Running in headless mode")
+                    except ImportError:
+                        print_error("pyvirtualdisplay is not installed in order "
+                                    "to launch the browser in headless mode")
+                        status = False
+                    except:
+                        print_error("Xvfb is not installed in order "
```

Comment:
Hi @brettleehari,

This line will cal every exception that occurs and will print out the statement on line 255 - which can result in a false exception detection. Please change lines 254, 255, and 256 to:

```
except Exception as e:
     print_error("An Exception Occurred: {0}".format(e))
```

Reply:
Assumption is line 246 would be caught by Import Error exception, line 247 & 248 would mostly hit exception when xvfb is not installed this is caught by the general exception :). line 249 exception is very unlikely but I agree with you comment that line 249 case makes this comment more solid. 
Thanks and changed.